cluster-provision/images/vm-image-builder/alpine-cloud-init/create-image.sh

```diff
@@ -24,7 +24,7 @@ docker run --rm --platform=$PLATFORM -v /lib/modules:/lib/modules -v /dev:/dev -
 ./alpine-make-vm-image \
 	--image-format qcow2 \
 	--image-size 200M \
-    --branch v3.16 \
```

Comment:
Can you please fix the alignment while you're at it? It really bugs me that this line doesn't sit flush with the surrounding ones O:-)

Reply:
sure, done